README.md

```diff
@@ -38,6 +38,17 @@ Nainstalovanou aplikaci spustíš následovně:
   ```
 * Program vypíše adresu (např. `http://0.0.0.0:8003/`); tu navštiv v prohlížeči.
 
+Vygenerovat HTML soubory (např. pro nahrání na statický hosting) lze takto:
+
+* Spusť freeze:
+  ```console
+  $ PYTHONPATH=. python -m naucse freeze
```

Comment:
V tomto souboru už jsou použité pipenv příkazy, které i zahrnují rovnou i tu proměnou prostředí. Takže stačí `pipenv run naucse freeze`. Na případné okamžitě spuštění serveru s obsahem lze pak přidat `--freeze` přepínač. Nicméně nejde to zpětně, ale chtělo by to asi zmínit...

Reply:
@mikicz Nebylo by možná lepší přidat `pipenv run freeze`?

Reply:
To asi jo...

Reply:
Jinak jsem se přepsal, přepínač je `--serve`, takže jako `pipenv run naucse freeze --serve`